sphinx/source/index.rst.txt

```diff
@@ -0,0 +1,45 @@
+.. _about:
```

Comment:
This file got saved with the wrong extension, it should be `index.rst` not `index.rst.txt`

Reply:
Whops, my bad. Will change that.

Reply:
This stray file happened to appear our of nowhere! Removed it.